training/eval/bucketise_predictions.py

```diff
@@ -0,0 +1,81 @@
+"""
+TODO: refactor to take arguments from command line instead of hardcoding
```

Comment:
for now, it can be left as is. In the future, let's put this file into the `scripts` directory and add `typer` to handle the arguments from cli